v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -23,13 +23,11 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
 
   @TemplateParameter.Text(
       order = 1,
-      optional = true,
       regexes = {"^.+$"},
       groupName = "Source",
       description = "Comma-separated Cloud Storage path(s) of the JDBC driver(s)",
       helpText = "The comma-separated list of driver JAR files.",
       example = "gs://your-bucket/driver_jar1.jar,gs://your-bucket/driver_jar2.jar")
-  @Default.String("")
   String getJdbcDriverJars();
```

Comment:
Please retain the default.

The default "" implies, a user has compiled the jars into the build via say mvn imports, so no need to external class loading.

This is done for say UT, where we use derby dependency from mvn.

Reply:
Done